src/SprykerEco/Zed/Econda/Business/Manager/CollectorManager.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Manager;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Shared\SqlCriteriaBuilder\CriteriaBuilder\CriteriaBuilderInterface;
+use SprykerEco\Zed\Econda\Business\Collector\DatabaseCollectorInterface;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\WriterInterface;
+use SprykerEco\Zed\Econda\Business\Helper\ProgressBarHelperInterface;
+use SprykerEco\Zed\Econda\Persistence\EcondaQueryContainerInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CollectorManager implements CollectorManagerInterface
+{
+    /**
+     * @var \Spryker\Shared\SqlCriteriaBuilder\CriteriaBuilder\CriteriaBuilderInterface
+     */
+    protected $criteriaBuilder;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Persistence\EcondaQueryContainerInterface
+     */
+    protected $queryContainer;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Business\Helper\ProgressBarHelperInterface
+     */
+    protected $progressBarHelper;
+
+    /**
+     * CollectorManager constructor.
```

Comment:
These comments are not required for `constructors`

Reply:
done